### ISSUE TITLE
[Suggestion] Add an optional whitelist or blacklist to the AutoLog module

### ISSUE THREAD

**Issue by supercellgamer** [MeteorDevelopment/meteor-client#2351]
### Describe the feature

Having a blacklist for players in the AutoLog module would be really useful. There is already the Only Trusted option, but that would require adding everyone to the friend whitelist to blacklist a few users. It would be much more convenient and useable if the feature was already in the client.

### Before submitting a suggestion

- [X] This feature doesn't already exist in the client.
(I have checked every module and their settings on the **latest dev build**)

- [X] This wasn't already suggested.
(I have checked every suggestion on github)

- [X] This feature is a valid suggestion.
(I have read the FAQs and Guides)

**Comment by Eccys** [comment]
yeah that would be less annoying having your friends show up all the time in notifs :)
